### ISSUE TITLE
Shouldn't `jake` be a devDependency?

### ISSUE THREAD

**Issue by kirillgroshkov** [mde/ejs#510]
Shouldn't jake be a devDependency?

**Comment by ben-tbotlabs** [comment]
I think it is because of the CLI that was added on Apr 19, 2020, so it is required to be a full dependency.

But to be clear in order to add this dependency you have added the following dependency tree

┬ ejs@3.1.2
│ └─┬ jake@10.6.1
│   ├── async@0.9.2
│   ├─┬ chalk@2.4.2
│   │ ├─┬ ansi-styles@3.2.1
│   │ │ └─┬ color-convert@1.9.3
│   │ │   └── color-name@1.1.3
│   │ ├── escape-string-regexp@1.0.5 deduped
│   │ └─┬ supports-color@5.5.0
│   │   └── has-flag@3.0.0
│   ├─┬ filelist@1.0.1
│   │ └── minimatch@3.0.4 deduped
│   └── minimatch@3.0.4 deduped

One of the major selling points of EJS IMO was no module dependencies, and would much rather have the CLI added a separate npm package then add these dependencies for a CLI we are not using

**Comment by kirillgroshkov** [comment]
I still think it was added by mistake

**Comment by mde** [comment]
Yes, Jake is currently a hard dependency of the new CLI. The desire for a separate CLI package is noted. I'll definitely consider it.

**Comment by RyanZim** [comment]
I'll throw in my vote for a separate CLI

**Comment by ryanburnette** [comment]
@mde I'll vote for anything that reduces dependencies.

**Comment by mourner** [comment]
I'd also like to encourage cutting out the CLI into a separate package (`ejs-cli`). Having no dependencies was one of the biggest advantages of this library.

**Comment by allnulled** [comment]
Yes, it would be very nice to not include jake + (filelist minimatch chalk async) + (...inner).

**Comment by zce** [comment]
I think it may be that jake needs a free advertising space, Just kidding :)

**Comment by sseide** [comment]
is there some update on the new ejs cli package to remove jake and all its dependencies from ejs again?

**Comment by mourner** [comment]
If anyone's looking for a modern EJS implementation with zero dependencies, check out https://github.com/mourner/yeahjs

**Comment by arthanzel** [comment]
Is the only reason that Jake is included in `cli.js` for argument parsing? It shouldn't be too difficult to write an inline parser for such a simple program. I hardly think that yet another NPM package is the best solution here.

**Comment by mde** [comment]
I'd be happy to consider a PR that implements this. It was just the fastest solution I had.

**Comment by arthanzel** [comment]
I'd be happy to contribute one

**Comment by allnulled** [comment]
I use ejs because it is the most intuitive solution for js templating I have found, and coding templates with it is really easy. I proposed already to separate ejs-cli from ejs, if ejs could be loaded as a single UMD script file from the web, it would also be incredibly powerful to be used in the web, at the level of jQuery even. But anyway, this is an author's decision.
